vars/artifactsPipeline.groovy

```diff
@@ -189,13 +189,12 @@ def call(Map pipelineParams) {
                                                 """
                                             }
                                             stage("Collect log data (${instance_type})") {
-                                                catchError(stageResult: 'FAILURE') {
-                                                    wrap([$class: 'BuildUser']) {
-                                                        dir('scylla-cluster-tests') {
-                                                            runCollectLogs(params, builder.region)
```

Comment:
The is correct, but the same problem exists in runCleanupResource(), runSendEmail().

Reply:
i have reverted it for all these functions as well @amoskong

Reply:
OK, I'm ok with this QUICK fix if we want to make the jobs back to stable ASAP.